modules/Students/applicationForm_manage_edit.php

```diff
@@ -175,23 +182,25 @@
     // PAYMENT
     $currency = getSettingByScope($connection2, 'System', 'currency');
     $applicationFee = getSettingByScope($connection2, 'Application Form', 'applicationFee');
+    $applicationProcessFee = getSettingByScope($connection2, 'Application Form', 'applicationProcessFee');
+    $applicationFee = getSettingByScope($connection2, 'Application Form', 'applicationFee');
```

Comment:
Does this not exist twice now, as it is two lines above?

Reply:
Right, well caught! Must have double-tapped the duplicate line command.